applications/nrf5340_audio/src/bluetooth/bt_stream/broadcast/broadcast_sink.c

```diff
@@ -363,7 +363,15 @@ static void base_recv_cb(struct bt_bap_broadcast_sink *sink, const struct bt_bap
 
 	if (suitable_stream_found) {
 		/* Set the initial active stream based on the defined channel of the device */
-		channel_assignment_get((enum audio_channel *)&active_stream_index);
+		enum audio_channel audio_channel_temp;
+
+		channel_assignment_get(&audio_channel_temp);
```

Comment:
`channel_value` that channel_assignment_get() uses to populate `audio_channel_temp` is a uint8_t, so this can never be larger than UINT8_MAX. Also, it should check against AUDIO_CH_NUM